Copy edits on JS3S1 (#432)
CodeYourFuture/curriculum: content/en/js3/blocks/using-map/index.md

We want to create a new array by applying a function to each element in the starting array. Earlier, we used a `for...of` statement to apply the function `createFilmCard` to each element in the array. However, we can also build an array using [the `map` array method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map). `map` is a {{<tooltip title="higher order function">}} A **higher-order function** is a function that takes another function as an argument or returns a new function{{</tooltip>}}. In this case, it means we pass a function as an argument to `map`. Then `map` will use this function to create a new array.

> Let's work through an example separate from our film data to make sure we understand `map` before we try to apply it to our film data:

Reviewer: ```suggestion
Work through this `map` exercise. It's important to understand map before we apply it to our film data.
```
